model/src/main/proto/javaclasses/mealorder/c/events.proto

```diff
@@ -37,265 +37,265 @@ import "javaclasses/mealorder/identifiers.proto";
 import "javaclasses/mealorder/model.proto";
 import "javaclasses/mealorder/values.proto";
 
-// An event reflecting the vendor addition.
+// The event reflecting a vendor addition.
 //
 message VendorAdded {
 
-    // An identifier of the added vendor.
+    // The identifier of a added vendor.
     VendorId vendor_id = 1;
 
-    // An identifier of the user who added the vendor.
+    // The identifier of the user who added a vendor.
     UserId who_added = 2;
 
-    // Time of addition.
+    // The time of addition.
```

Comment:
time - uncountable noun when we are talking about things in general.
That it's why without articles.